src/foam/u2/dialog/ApplicationPopup.js

```diff
@@ -210,7 +215,7 @@ foam.CLASS({
             .call(function() { content = this; })
           .end()
           .start()
-            .addClasses([this.myClass('footer'), 'p-legal-light'])
+            .addClasses([this.myClass('footer'), 'p-legal-light']).show(this.displayFooter)
```

Comment:
Should be automatically hidden if the footerString is empty, then we dont need the extra boolean